backend/danswer/bots/slack/listener.py

```diff
@@ -264,35 +270,40 @@ def process_message(
     engine = get_sqlalchemy_engine()
     with Session(engine) as db_session:
         slack_bot_config = get_slack_bot_config_for_channel(
-            channel_name=channel_name, db_session=db_session
+            channel_name=channel_name or "Private Channel", db_session=db_session
```

Comment:
nit: maybe add a comment about how we know Private Channel could never overlap a real channel name